articles/getting-started-with-game-development-using-windows-forms/index.md

```diff
@@ -0,0 +1,320 @@
+The method will take the 4 colors chosed by the player the 4 colors produced by the system and the 4 hint digits as parameters.
+
+
+The first condition will check if the 4 color digits are fully clicked by the user, if yes it will turn the second Check button on, if not it will let the player know that he hasn't picked 4 colors in order to finish the round.
+```c#
+ void Check_button(Button Col1, Button Col2, Button Col3, Button Col4,
+                   Button cool1, Button cool2, Button cool3, Button cool4,
+                   Button Show1, Button Show2, Button Show3, Button Show4)
+        {
+            int q = 0;
+            int w = 0;
+            VIC.Text = "";
+            if (Col1.BackColor != Color.Gainsboro && Col2.BackColor != Color.Gainsboro && Col3.BackColor != Color.Gainsboro && Col4.BackColor != Color.Gainsboro)
+                ch2.Visible = true;
+            else
+                VIC.Text = "ENTER SOME COLORS DUMMY";
+```
+
+The next piece of code will increase the (q) counter by 1 each time it finds one of the colors chosed by the player in one of the digits produced by the system (but not in the right place). 
+
+```c#
+            if (Col1.BackColor == cool1.BackColor || Col1.BackColor == cool2.BackColor || Col1.BackColor == cool3.BackColor || Col1.BackColor == cool4.BackColor)
+                q++;
+            if (Col2.BackColor == cool1.BackColor || Col2.BackColor == cool2.BackColor || Col2.BackColor == cool3.BackColor || Col2.BackColor == cool4.BackColor)
+                q++;
+            if (Col3.BackColor == cool1.BackColor || Col3.BackColor == cool2.BackColor || Col3.BackColor == cool3.BackColor || Col3.BackColor == cool4.BackColor)
+                q++;
+            if (Col4.BackColor == cool1.BackColor || Col4.BackColor == cool2.BackColor || Col4.BackColor == cool3.BackColor || Col4.BackColor == cool4.BackColor)
+                q++;
+```
+
+This counter will decide how many red buttons will glow from the 4 hint buttons.
+
+```c#
+if (q == 1)
+                Show1.BackColor = Color.Red;
+            else if (q == 2)
+            {
+                Show1.BackColor = Color.Red;
+                Show2.BackColor = Color.Red;
+            }
+            else if (q == 3)
+            {
+                Show1.BackColor = Color.Red;
+                Show2.BackColor = Color.Red;
+                Show3.BackColor = Color.Red;
+            }
+            else if (q == 4)
+            {
+                Show1.BackColor = Color.Red;
+                Show2.BackColor = Color.Red;
+                Show3.BackColor = Color.Red;
+                Show4.BackColor = Color.Red;
+            }
+```
+The next piece of code will check how many buttons chosed by the player that has the same **color** and **digit** produced by the system. For an example the first line of code will give the w counter the number 3 if 3 of the buttons chosed by the player has the same color and digit produced by the system and so on.

```

Comment:
chosed to chosen.

This statement should read

The next piece of code will check how many buttons chosen by the player that has the same **color** and **digit** produced by the system. For example, the first line of code will give the w counter the number 3 if 3 of the buttons chosen by the player have the same color and digit produced by the system and so on.